src/SWP/Bundle/FixturesBundle/DataFixtures/PHPCR/LoadMenuNodesData.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: sourcefabric
+ * Date: 14/07/16
+ * Time: 13:45
+ */
+
+namespace SWP\Bundle\FixturesBundle\DataFixtures\PHPCR;
+
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use SWP\Bundle\FixturesBundle\AbstractFixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Cmf\Bundle\MenuBundle\Doctrine\Phpcr\MenuNode;
+
+class LoadMenuNodesData extends AbstractFixture implements FixtureInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(ObjectManager $manager)
+    {
+        $env = $this->getEnvironment();
+        $menuNodes = [
+            'dev' => [
```

Comment:
yes, add new widget with menu widget type to container with demo data - so after installation of default theme we will have it visible.